content/ctl_automate_server.md

```diff
@@ -1383,8 +1382,6 @@ the **opscode-erchef** should be investigated further:
 run: oc-id
 run: opscode-chef: (pid 4327) 13671s; run: log: (pid 4326) 13671s
 run: opscode-erchef: (pid 5383) 5s; run: log: (pid 4382) 13669s
```

Comment:
@PrajaktaPurohit Would you like references to `opscode-erchef` removed also?

Reply:
@mjingle Not in this case.

Reply:
@mjingle these are OK